### PR TITLE
ISPN-13992-a: Document cache rebalancing metrics

### PR DESCRIPTION
**Pull request by dfitzmau**
Jira: https://issues.redhat.com/browse/ISPN-13992

### CODE REVIEW

**Comment by dfitzmau** [comment]
Hi @dvagnero and @jabolina . I experienced with file addition issues with https://github.com/infinispan/infinispan/pull/10211 , so I moved the content to PR #10268.

You OK to approve this PR?

**Review by dvagnero — CHANGES_REQUESTED** [review]

File: documentation/src/main/asciidoc/topics/proc_exporting_metrics_state_transfer.adoc
Comment (dvagnero) [reply documentation/src/main/asciidoc/topics/proc_exporting_metrics_state_transfer.adoc:19]: Where do you see that embedded or remote are cache types?
We should be consistent and use the same terminology and the same applies to statistics and metrics. Go with the terminology that is used

File: documentation/src/main/asciidoc/topics/proc_exporting_metrics_state_transfer.adoc
Comment (dvagnero) [inline documentation/src/main/asciidoc/topics/proc_exporting_metrics_state_transfer.adoc:32]: Consistency & Jupiter requirements 
No plain paragraphs in the additional resources section. We no longer use this format in our docs.

```suggestion
[role="_additional-resources"]
.Additional resources
* link:{config_docs}#statistics-jmx[Enabling and configuring {brandname} statistics and JMX monitoring]
* link:{javadocroot}/jmxComponents.html#StateTransferManager[`StateTransferManager` ({brandname} {schemaversion} API)]
```

**Review by dvagnero — CHANGES_REQUESTED** [review]

File: documentation/src/main/asciidoc/topics/proc_exporting_metrics_state_transfer.adoc
Comment (dfitzmau) [reply documentation/src/main/asciidoc/topics/proc_exporting_metrics_state_transfer.adoc:19]: "Cache types" is not a specific term I'm trying to specify. I am speaking in a generalized tone to refer to embedded caches and remote caches. 

My only suggestion is:

> Enable metrics for either your embedded cache or yoru remote cache.

WDYT? Originally, I felt I needed to add "type" to avoid a misinterpretation that embedded and remote caches were metrics.

Comment (dvagnero) [reply documentation/src/main/asciidoc/topics/proc_exporting_metrics_state_transfer.adoc:19]: Enable metrics for your clustered cache?

Comment (dfitzmau) [reply documentation/src/main/asciidoc/topics/proc_exporting_metrics_state_transfer.adoc:19]: Clustered caches might be mislead because the following document section explains clustered caches as replicated, distributed, and so on:

[Clustered caches](https://infinispan.org/docs/stable/titles/configuring/configuring.html#clustered-caches)

File: documentation/src/main/asciidoc/topics/proc_exporting_metrics_state_transfer.adoc
Comment (dfitzmau) [reply documentation/src/main/asciidoc/topics/proc_exporting_metrics_state_transfer.adoc]: Changed link format to match current Jupiter styling.

File: documentation/src/main/asciidoc/topics/proc_exporting_metrics_state_transfer.adoc
Comment (dvagnero) [reply documentation/src/main/asciidoc/topics/proc_exporting_metrics_state_transfer.adoc:19]: right, those are cache modes https://infinispan.org/docs/stable/titles/configuring/configuring.html#cache-mode-comparison_caches 
But this is fine

File: documentation/src/main/asciidoc/topics/proc_exporting_metrics_state_transfer.adoc
Comment (dvagnero) [reply documentation/src/main/asciidoc/topics/proc_exporting_metrics_state_transfer.adoc]: Thanks. Can you also remove the "." in the end?

Comment (dfitzmau) [reply documentation/src/main/asciidoc/topics/proc_exporting_metrics_state_transfer.adoc]: Removed.

**Comment by dfitzmau** [comment]
> LGTM

Hi @dvagnero . Is the Approval button optional? Must I get two to three approvals before the PR can be merged? Some upstream repos have this setting in place; not sure if Infinispan is the same?

**Comment by dvagnero** [comment]
I think you need one SME to approve or comment that the PR is correct so it's clear that the SME review was done.
For trivial doc fixes it should be enough to have an approval from another docs team member.

**Review by jabolina — APPROVED** [review]

**Comment by jabolina** [comment]
Merged, thanks @dfitzmau @dvagnero

**Pull request merged**
